docs/kratos/session-management/01_overview.mdx

```diff
@@ -27,113 +30,82 @@ For security reasons, you can't break the isolation between cookies and session
 
 ## Ory Session
 
-This is an sample Ory Session:
-
-```json
-{
-  "id": "1338410d-c473-4503-a96a-53efa06e2531",
-  "active": true,
-  "expires_at": "2021-10-15T15:58:57.683338Z",
-  "authenticated_at": "2021-10-14T15:58:57.683338Z",
-  "authenticator_assurance_level": "aal2",
-  "authentication_methods": [
-    {
-      "method": "password",
-      "completed_at": "2021-10-14T15:55:19.03621Z"
-    },
-    {
-      "method": "lookup_secret",
-      "completed_at": "2021-10-14T15:56:21.257867Z"
-    },
-    {
-      "method": "lookup_secret",
-      "completed_at": "2021-10-14T15:58:57.683337Z"
-    },
-    {
-      "method": "lookup_secret",
-      "completed_at": "2021-10-14T16:03:14.833192Z"
-    }
-  ],
-  "issued_at": "2021-10-14T15:58:57.683338Z",
-  "identity": {
-    "id": "9496bbd5-f426-473f-b087-c7df853f274a",
-    "schema_id": "default",
-    "schema_url": "https://<your-project-slug>.projects.oryapis.com/schemas/default",
-    "state": "active",
-    "state_changed_at": "2021-10-14T17:55:17.885497+02:00",
-    "traits": {
-      "website": "https://www.ory.sh/",
-      "email": "0.wz4yhr0zwyd@ory.sh"
-    },
-    "verifiable_addresses": [
-      {
-        "id": "4de14270-ca13-4efb-ac3f-8f03b1f649d8",
-        "value": "0.wz4yhr0zwyd@ory.sh",
-        "verified": false,
-        "via": "email",
-        "status": "sent",
-        "created_at": "2021-10-14T17:55:17.886639+02:00",
-        "updated_at": "2021-10-14T18:03:14.839009+02:00"
-      }
-    ],
-    "recovery_addresses": [
-      {
-        "id": "fdab5a5f-5efc-4202-93b5-fd3ee632420b",
-        "value": "0.wz4yhr0zwyd@ory.sh",
-        "via": "email",
-        "created_at": "2021-10-14T17:55:17.886831+02:00",
-        "updated_at": "2021-10-14T18:03:14.839105+02:00"
-      }
-    ],
-    "created_at": "2021-10-14T17:55:17.886237+02:00",
-    "updated_at": "2021-10-14T17:55:17.886237+02:00"
-  }
-}
-```
+An example of an Ory session can be seen in the [Get session API](../../reference/api#tag/identity/operation/disableSession)
```

Comment:
Why `disableSession` instead of `listSessions`?
https://docs-btk6arvlq-ory.vercel.app/docs/reference/api#tag/identity/operation/listSessions

Reply:
fixed to `getSession`